content/applications/sales/sales/products_prices/returns.rst

```diff
@@ -0,0 +1,100 @@
+===========================
+Process returns and refunds
+===========================
+
+The :guilabel:`Sales` app provides two different ways to process returns based on whether an
+invoice has been sent or not.
+
+Before invoicing: Reverse Transfers
+===================================
+
+Returns are completed using *Reverse Transfers* when a customer decides to return a product before
+an invoice has been sent or validated.
+
+.. note::
+   In order to use Reverse Transfers, the :guilabel:`Inventory` app must also be installed.
+
+To start a return, navigate to the customer's sales order, and click on the :guilabel:`Delivery`
+smart button to open the associated delivery order.
+
+On the validated delivery order, click :guilabel:`Return` to open the :guilabel:`Reverse Transfer`
+pop-up window. By default, the :guilabel:`Quantity` matches the validated quantities from the
+delivery order. Update the quantities if necessary. Click on the trash icon next to a line item to
+remove it from the return.
+
+.. image:: returns/reverse-transfer-popup.png
+   :align: center
+   :alt: Reverse Transfer Pop-Up
+
+Click :guilabel:`Return` to confirm the return. This generates a new warehouse operation for the
+incoming returned product(s). Upon receiving the return, the warehouse team *validates* the
+warehouse operation. Then, on the original sales order, the :guilabel:`Delivered` quantity will
+reflect the difference between the initial validated quantities and the returned quantities.
+
+.. image:: returns/updated-sales-quantities.png
+   :align: center
+   :alt: Updated Sales Order Quantities
+
+When an invoice is created, the customer receives an invoice only for the products they are
+keeping.
+
+After invoicing
+===============
+
+Sometimes, customers receive and/or pay their invoice before returning an item. In these cases, a
```

Comment:
Rewording to be less confusing since this section is called "After invoicing"
```suggestion
Sometimes, customers return an item after they receive and/or pay for their invoice. In these cases, a
```